concordium-std/src/impls.rs

```diff
@@ -516,10 +516,17 @@ where
     pub fn into_key(self) -> K { self.key }
 
     /// Sets the value of the entry with the `VacantEntry`'s key.
-    pub fn insert(mut self, value: V) {
+    pub fn insert(mut self, value: V) -> OccupiedEntry<'a, K, V, StateEntryType> {
         // Writing to state cannot fail.
+        // TODO: This is subtle. Should we normalize to 0 first, and reset the cursor?
```

Comment:
I assume this means whether we should overwrite any existing values with 0s.
I don't think that is necessary, because the only way to get a `VacantEntry` is through `StateMap::entry`, which calls `prims::create_state_entry`, which in turn ensures that we are looking at a blob of bytes.